public/css/team.css

```diff
@@ -1,70 +1,82 @@
-.header {
-	visibility: hidden;
-	margin: 0px;
-	width: 0;
+/* Header */
+.logo-header {
+	display: flex;
+	align-items: center;
+	justify-content: center;
+}
+
+.logo-img-header {
+	position: absolute;
+	top: 170px;
 }
 
-.directors-header {
-	margin: 50px 0px 40px 0px;
-	display: block;
-	max-width: 65%;
-	display: inline-block;
+.logo-img {
+	margin-left: 50%;
+	transform: translateX(-50%);
+	max-width: 100%;
+}
+
+.logo-txt {
+	display: inline;
+	color: white;
+	position: absolute;
+	margin-left: 50%;
+	transform: translateX(-50%);
+	top: 70px;
+	left: 120px;
+	right: -80px;
+	font-size: 34px;
+	font-weight: bold;
 }
 
 .img-header {
 	width: 100%;
+	min-width: 800px;
 }
```

Comment:
So I said I was fine with cutting off part of the graphic before, but I think it would actually be better if we didn't. If you give the main graphic a 66px top margin (the width of the navbar), it'll have enough room in the sky so that the logo doesn't overlap with the mountains on small screens even without the `min-width`.

I will point out, though, this makes the black tab that says "Team Captains" that you've been hiding start overlapping with the bottom of the graphic on small screens. Not sure what the best way to address that is; might be to just manually place the section header and not use `.black-tab-container`.

Reply:
If it changes anything, I talked to courtney yesterday about it (before reading this lol) and she said she didn't care about cutting off the hikers. I'll still modify it to how you want though.